runtime/t2rn-parachain/src/precompiles.rs

```diff
@@ -0,0 +1,57 @@
+use crate::Runtime;
+
+pub fn generate_precompile_set(
+) -> sp_std::vec::Vec<(sp_core::H160, evm_precompile_util::KnownPrecompile<Runtime>)> {
+    // Generate intiial precopile set
+    let mut precompile_set = sp_std::vec![
+        (
+            sp_core::H160([0u8; 20]),
+            evm_precompile_util::KnownPrecompile::ECRecover
+        ),
+        (
+            sp_core::H160([1u8; 20]),
+            evm_precompile_util::KnownPrecompile::Sha256
+        ),
+        (
+            sp_core::H160([2u8; 20]),
+            evm_precompile_util::KnownPrecompile::Ripemd160
+        ),
+        (
+            sp_core::H160([3u8; 20]),
+            evm_precompile_util::KnownPrecompile::Identity
+        ),
+        (
+            sp_core::H160([4u8; 20]),
+            evm_precompile_util::KnownPrecompile::Modexp
+        ),
+        (
+            sp_core::H160([5u8; 20]),
+            evm_precompile_util::KnownPrecompile::Sha3FIPS256
+        ),
+        (
+            sp_core::H160([6u8; 20]),
+            evm_precompile_util::KnownPrecompile::Sha3FIPS512
+        ),
+        (
+            sp_core::H160([7u8; 20]),
+            evm_precompile_util::KnownPrecompile::ECRecoverPublicKey
+        ),
+        (
+            sp_core::H160([8u8; 20]),
+            evm_precompile_util::KnownPrecompile::Portal
+        ),
+        (
+            sp_core::H160([9u8; 20]),
```

Comment:
This address overlaps with TokenPrecompile addresses and that will produce errors... We could just route 
it to the desired address in the new T3rnPrecompiles setup instead...

Reply:
Fixed in [`271c1ed` (#1566)](https://github.com/t3rn/t3rn/pull/1566/commits/271c1edc2264e4eb56949cd7b0851f0d5530c3c9)